### DISCUSSION TITLE
Measurement Point Label Values

### DISCUSSION THREAD

**Discussion by KorecGroup-AliJafri** [zouyaoji/vue-cesium#581] [Q&A] [answered]
Hi! I wanted to ask, is it possible to add an additional value or inject text inside the label text that appears when a point using the measuring tools? If not then what would be the ideal solution? Via Cesium API?

**Comment by zouyaoji** [comment, marked as answer]
Via Cesium API

![image](https://github.com/user-attachments/assets/9860ed28-7b05-461d-ac2e-75fee6af48e1)

**Reply by KorecGroup-AliJafri** [reply]
Great, thank you!

**Reply by PetukhovArt** [reply]
> Great, thank you!

hi , you understood how to change this after mount ? Can you give some advices

**Comment by KorecGroup-AliJafri** [comment]
You can pass the emitted values as shown in console example into variables and manipulate them
